chainer/functions/activation/leaky_relu.py

```diff
@@ -37,6 +42,17 @@ def forward_cpu(self, inputs):
             self.retain_inputs((0,))
         return y,
 
+    def forward_ideep(self, inputs):
```

Comment:
Could you add test case for this?

Reply:
will provide it later

Reply:
test cases have been added, please check.